src/pkg/inspection/controller.go

```diff
@@ -331,6 +333,21 @@ func (c *controller) Run(ctx context.Context, policy *v1alpha1.InspectionPolicy)
 		}
 	}
 
+	// Read config from InspectionPolicy, send assessment reports to Governor api if governor enabled.
+	if policy.Spec.Inspection.Assessment.Governor.Enabled {
+		governorConfig := policy.Spec.Inspection.Assessment.Governor
+
+		if governorConfig.ClusterID == "" || governorConfig.URL == "" || governorConfig.APIToken == "" {
+			log.Error("Either ClusterID or URL or APIToken is empty")
```

Comment:
Log messages have to be more specific indicating GovernorExporter.

Rephrase all the error messages with the `Governor` prefix
log.Errorf("GovernorExporter : Error message")